gm4_heart_canisters/data/heart_canisters/functions/check.mcfunction

```diff
@@ -0,0 +1,11 @@
+scoreboard players set @s[nbt={Inventory:[{Count:1b,tag:{gm4_heart_canister_tier:1b}}]}] heart_can_health 1
```

Comment:
This function is to vaguely named. "heart_canister_count" or "check_heart_cans" would be more descriptive